SAMv1.tex

```diff
@@ -413,7 +415,8 @@ \subsection{The alignment section: mandatory fields}\label{sec:alnrecord}
   \item Bit 0x10 indicates whether {\sf SEQ} has been reverse complemented
     and {\sf QUAL} reversed.
     When bit~0x4 is unset, this corresponds to the strand to which the
-    segment has been mapped.
+    segment has been mapped: bit~0x10 unset indicates the forward strand,
+    while set indicates the reverse strand.
```

Comment:
This is not consistent with the paragraph above: "If 0x4 is set, no assumptions can be made about {\sf
      RNAME}, {\sf POS}, {\sf CIGAR}, {\sf MAPQ}, and bits 0x2, 0x100,  and 0x800".

That implies when 0x4 is set, 0x10 still has meaning.  My recollection is that this was previously also listed as an undefined case but it was changed in order to permit SAM to FASTQ conversion to work with placed-but-unmapped reads without making assumptions on undefined fields.  If we have a requirement that 0x4 being set is that the sequence is in the original orientation as it came off the instrument, then it should be stated as such - eg if 0x4 is set then the 0x10 flag must be unset.  Instead it's probably easier just to remove any statements about 0x10 only having meaning when 0x4 is unset, in the knowledge that practically speaking it'll just be 0 anyway for the unmapped data so has no impact.

Apart from this rather old problem, which isn't a fault of this PR itself, I'm happy with it.

Reply:
Yep, the following sentence is closely related :smile:.

We would love to be able to just say 0x4 set ⇒ 0x10 unset, but all this verbiage is to allow for the BWA bug whereby some reads that end up being output as unmapped have already been revcomped internally, and BWA just outputs them that way around. This was #52.

Reply:
Ah yes that was it.  I was trying to remember why the orientation was still important knowledge for the unmapped data.  Far too much of SAM has been specified in order to work around bwa!